pages/gallery.js

```diff
@@ -0,0 +1,44 @@
+import { useState } from "react";
+import { Gallery } from "react-grid-gallery";
+import Lightbox from "react-image-lightbox";
+import "react-image-lightbox/style.css";
+import images from "./ass/images";
+
+export default function Gallery() {
```

Comment:
Change to `GalleryPage`.. 'Gallery' is contradicting to the import and the build is failing. Thought you'll find the error before pushing